rspace/src/main/scala/coop/rchain/rspace/PointerBlock.scala

```diff
@@ -0,0 +1,31 @@
+package coop.rchain.rspace
+
+import scodec.Codec
+import scodec.Codec._
+import scodec.codecs._
+
+case class PointerBlock(toVector: Vector[Option[Blake2b256Hash]]) {
+
+  require(toVector.length == PointerBlock.length)
+
+  def updated(tuples: List[(Int, Option[Blake2b256Hash])]): PointerBlock =
+    PointerBlock(tuples.foldLeft(toVector) { (vec, curr) =>
+      vec.updated(curr._1, curr._2)
+    })
+
+  def children: Vector[(Int, Blake2b256Hash)] =
+    toVector.zipWithIndex.collect { case (Some(hash), idx) => (idx, hash) }
+}
+
+object PointerBlock {
+
+  val length = 256
```

Comment:
Why do you need 256 hashes preallocated ? It's still possible to create instance of `PointerBlock` of different length.

Reply:
Is it?

Reply:
No I cannot because of `require` condition.